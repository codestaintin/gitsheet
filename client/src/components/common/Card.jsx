import React, { Fragment } from 'react';

const Card = () => (
  <Fragment>
    <div className="card border-primary">
      <div className="card-header">Category</div>
      <div className="card-body">
        <p className="card-text">
        This is a longer card with supporting text
        below as a natural lead-in to additional content.
        This content is a little bit longer.
        </p>
      </div>
    </div>
  </Fragment>
);

export default Card;