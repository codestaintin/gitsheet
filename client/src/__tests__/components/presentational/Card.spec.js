import React from 'react';
import { shallow } from 'enzyme';
import Card from '../../../components/common/Card';

describe('<Card>', () => {
  const props = {
    name: '',
    gitCheats: []
  };
  it('should render the Card component', () => {
    const tree = shallow(<Card {...props} />);
    expect(tree).toMatchSnapshot();
  });
});