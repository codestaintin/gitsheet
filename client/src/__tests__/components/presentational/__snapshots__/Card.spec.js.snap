// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Card> should render the Card component 1`] = `
<Fragment>
  <div
    className="card border-light"
  >
    <div
      className="card-header capitalise"
    >
      <b />
    </div>
  </div>
</Fragment>
`;
